stable/mysqldump/values.yaml

```diff
@@ -3,8 +3,8 @@
 # Declare variables to be passed into your templates.
 
 image:
-  repository: mysql
-  tag: "5.7.21"
+  repository: monotek/gcloud-mysql
+  tag: "6"
```

Comment:
It's unclear which gcloud sdk and mysql-client version is used here.

Reply:
For the user it is still not clear which gcloudSDK and mysql client is packaged in this docker image. So it's actually an issue of the docker image itself.